src/main/java/org/avarion/graves/Graves.java

```diff
@@ -1,14 +1,68 @@
 
-import com.google.common.base.Charsets;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
+
 import org.avarion.graves.command.GravesCommand;
 import org.avarion.graves.command.GraveyardsCommand;
 import org.avarion.graves.compatibility.Compatibility;
 import org.avarion.graves.compatibility.CompatibilityBlockData;
-import org.avarion.graves.listener.*;
-import org.avarion.graves.manager.*;
+import org.avarion.graves.listener.BlockBreakListener;
+import org.avarion.graves.listener.BlockExplodeListener;
+import org.avarion.graves.listener.BlockFromToListener;
+import org.avarion.graves.listener.BlockPistonExtendListener;
+import org.avarion.graves.listener.BlockPlaceListener;
+import org.avarion.graves.listener.CreatureSpawnListener;
+import org.avarion.graves.listener.EntityDamageByEntityListener;
+import org.avarion.graves.listener.EntityDeathListener;
+import org.avarion.graves.listener.EntityExplodeListener;
+import org.avarion.graves.listener.HangingBreakListener;
+import org.avarion.graves.listener.InventoryClickListener;
+import org.avarion.graves.listener.InventoryCloseListener;
+import org.avarion.graves.listener.InventoryDragListener;
+import org.avarion.graves.listener.InventoryOpenListener;
+import org.avarion.graves.listener.PlayerBucketEmptyListener;
+import org.avarion.graves.listener.PlayerDeathListener;
+import org.avarion.graves.listener.PlayerDropItemListener;
+import org.avarion.graves.listener.PlayerInteractAtEntityListener;
+import org.avarion.graves.listener.PlayerInteractEntityListener;
+import org.avarion.graves.listener.PlayerInteractListener;
+import org.avarion.graves.listener.PlayerJoinListener;
+import org.avarion.graves.listener.PlayerMoveListener;
+import org.avarion.graves.listener.PlayerQuitListener;
+import org.avarion.graves.listener.PlayerRespawnListener;
+import org.avarion.graves.manager.BlockManager;
+import org.avarion.graves.manager.CacheManager;
+import org.avarion.graves.manager.DataManager;
+import org.avarion.graves.manager.EntityDataManager;
+import org.avarion.graves.manager.EntityManager;
+import org.avarion.graves.manager.GUIManager;
+import org.avarion.graves.manager.GraveManager;
+import org.avarion.graves.manager.GraveyardManager;
+import org.avarion.graves.manager.HologramManager;
+import org.avarion.graves.manager.ImportManager;
+import org.avarion.graves.manager.IntegrationManager;
+import org.avarion.graves.manager.ItemStackManager;
+import org.avarion.graves.manager.LocationManager;
+import org.avarion.graves.manager.RecipeManager;
+import org.avarion.graves.manager.VersionManager;
 import org.avarion.graves.type.Grave;
-import org.avarion.graves.util.*;
+import org.avarion.graves.util.FileUtil;
```

Comment:
Can you revert this back to the more succinct `import org.avarion.graves.util.*;`?
+ Also the others that are changed?

I know that IDEs have a habbit of doing that, and I know a `*` import isn't all that good, but in this case I favor it.